### DISCUSSION TITLE
Menu Bar Padding?

### DISCUSSION THREAD

**Discussion by navtoj** [jordanbaird/Ice#118] [Q&A] [answered]
I'm trying to replicate the menu bar shown below.

1. Is there a way to have padding around the menu bar?

2. Or, can the border color be transparent so that it can _act_ as "padding" around the menu bar?

![Klack Menu Bar](https://github.com/jordanbaird/Ice/assets/20136585/baaca172-a158-4453-a0c5-1c5b3f9f0e7f)
*Source: https://twitter.com/henrikruscon/status/1786683119262687278*

#### Bar Setup

- Shape Kind: `Split`
- Edges Type: `Rounded`

**Comment by jordanbaird** [comment, marked as answer]
This is only possible if you have a MacBook with the notch, as they have larger menu bars. macOS doesn't allow apps to change the size, unfortunately.

**Reply by navtoj** [reply]
Doesn't the below image from the readme page show that it's already possible though?

I'm not looking to increase the height of the bar. I was only wondering if it was possible to add some spacing around the bar so that it doesn't "touch" the edge of the screen.

![image](https://github.com/jordanbaird/Ice/assets/20136585/bef62af2-1d2a-4c6c-b1e8-2710a8fa88a3)

**Reply by jordanbaird** [reply]
The image in the readme was taken on a Mac that doesn't have a notch, so it uses the smaller menu bar. As you can see in the image, it still touches the edges of the screen. For what it's worth, the video in the tweet you linked is a mockup, and not actually from macOS.

**Reply by navtoj** [reply]
In version `0.11.5`, enabling the `Use inset shape on screens with notch` option achieves the look on my notched macbook.

![Screenshot 2024-10-20 at 2 38 13 PM](https://github.com/user-attachments/assets/634b1dc9-0993-4e1a-94b4-7eafcad1fc11)

**Reply by jordanbaird** [reply]
@navtoj So is there anything more to do here?

**Reply by navtoj** [reply]
No, I just wanted to post what worked for me in case someone else comes across this discussion later on.
